### PR TITLE
PPM Pickup and delivery locations handlers/model update [#156770787]

### PR DESCRIPTION
**Pull request by macrael**
## Description

This pr adds a few new fields to the PPM model and updates it's handler to accept them.

It also updates the PPM, Moves, and SignedCert handlers to use the modern style. 

## Code Review Verification Steps

* [ ] All tests pass.
* [ ] Code follows the guidelines for [Logging](https://github.com/transcom/mymove/blob/master/docs/backend.md#logging)
* [ ] The requirements listed in
 [Querying the Database Safely](https://github.com/transcom/mymove/blob/master/docs/backend.md#querying-the-database-safely)
 have been satisfied.
* [ ] There are no [aXe](https://www.deque.com/products/aXe/) warnings for UI.
* [ ] This works in IE.
* [ ] Any new third party client side dependencies (e.g., login.gov, google analytics, CDN libraries, etc.) have been communicated to @willowbl00.
* [ ] Request review from a member of a different team.
* [ ] (TRIAL) Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/156770787) for this change

### CODE REVIEW

File: pkg/handlers/converters.go [hunk 1]
```diff
@@ -34,6 +34,13 @@ func fmtDate(date time.Time) *strfmt.Date {
 	return &fmtDate
 }
 
+func fmtDatePtr(date *time.Time) *strfmt.Date {
```
Comment (tinyels) [inline pkg/handlers/converters.go]: how are we handling this for orders (which also have a date)?

File: pkg/handlers/handlers_test.go [hunk 2]
```diff
@@ -46,6 +47,7 @@ func (suite *HandlerSuite) checkErrorResponse(resp middleware.Responder, code in
 	errResponse, ok := resp.(*errResponse)
 	if !ok || errResponse.code != code {
 		suite.T().Errorf("Expected %s Response: %v", name, resp)
+		debug.PrintStack()
```
Comment (tinyels) [inline pkg/handlers/handlers_test.go]: did you intentionally leave this? Since it is a test, I think it could be useful

Comment (macrael) [reply pkg/handlers/handlers_test.go]: Yes, the tests were not printing a useful line number when they errored here so I think the stack trace will help.

File: pkg/handlers/converters.go (hunk 1 above)
Comment (macrael) [reply pkg/handlers/converters.go]: only difference is that the orders' date is not a pointer because it's required at the model level.

**Pull request merged**